Kernel/Arch/x86_64-pc/System/GDTFarJump.asm

```diff
@@ -1,5 +1,6 @@
 [bits 64]
 [global reloadSegments]
+[global set_up_stack]
```

Comment:
You should probably remove this line as there's no function called "set_up_stack" and no place where we use it

Reply:
Yeah, It was for some 32 bit things. I forgot to remove It, thanks